dockers/docker-snmp/snmpd.conf.j2

```diff
@@ -13,16 +13,10 @@
 #  AGENT BEHAVIOUR
 #
 
-#  Listen for connections on all ip addresses, including eth0, ipv4 lo
+#  Listen for connections on all ip addresses, including eth0, ipv4 and ipv6 lo
 #
-{% if SNMP_AGENT_ADDRESS_CONFIG %}
-{% for (agentip, port, vrf) in SNMP_AGENT_ADDRESS_CONFIG %}
```

Comment:
>SNMP_AGENT_ADDRESS_CONFIG [](http://example.com/codeflow?start=31&length=25)

This feature is still useful, and there are some other PRs trying to extend it.

Reply:
I am sorry, I wasn't aware of the other PR's for this feature.
The logic of address/port/vrf was moved to the docker create port and address forwarding. Since only user defined ports and addresses will be forwarded, this logic will no longer be relevant here

Reply:
I have updated the code to include the latest merged PR

Reply:
Could you explain "The logic of address/port/vrf was moved to the docker create port and address forwarding"?

Reply:
In the current implementation, snmpd has its configuration in snmpd.conf which defines which addresses and ports can be used for query, how and if vrf is being used, etc.
Removing snmp container from host network means that it is not exposed to any of these addresses by default,
What we did here was to use docker address:port forwarding as the method to implement the logic for it. When a user configures address and port to be used for snmp queries - this address:port tuple will be used when snmp docker is created, similar to a firewall. In what we offer here, the snmp demon inside the container will listen to all packets, and the docker networking logic (address:port forwarding) will forward only the relevant packets, as configured by user